wiki/en/en-Installation-for-Windows.md

```diff
@@ -9,7 +9,7 @@ permalink: "/wiki/Installation-for-Windows"
 
 # Installation for Windows
 Make sure you read the [Getting Started](Getting-Started) page.
-1. **Download and install an ASIO Driver**. It is recommended to use a sound card/interface with a native ASIO driver. If you don't have an external sound card, you probably need to install [this free ASIO driver (ASIO4ALL)](https://www.asio4all.org){: target="_blank" rel="noopener noreferrer"} before installing Jamulus.
+1. **Download and install an ASIO Driver**. It is recommended to use a sound card/interface with a native ASIO driver. If you don't have an external sound card, you probably need to install a generic driver like [ASIO4ALL](https://www.asio4all.org){: target="_blank" rel="noopener noreferrer"}. **Note:** Do not fall for  ads! Download the ASIO4ALL installer by scrolling down a bit! For more information scroll down to the [ASIO](#asio) section.
```

Comment:
Two spaces between "fall for" and "ads!" - should only be one.

Reply:
Done.